build-support/pants_venv

```diff
@@ -11,26 +11,52 @@ REQUIREMENTS=(
   ${REPO_ROOT}/pants-plugins/3rdparty/python/requirements.txt
 )
 
+venv_dir_prefix="${REPO_ROOT}/build-support/pants_dev_deps"
+
+function determine_specific_py3_version() {
+  version_number="$(${PY} -c 'import sys; print(sys.version_info[1])')"
+  case "${version_number}" in
+    6) echo "36" ;;
+    7) echo "37" ;;
+    8) echo "38" ;;
+    *) die "Pants cannot run with Python 3.${version_number}." ;;
+  esac
+}
```

Comment:
The reason we need this function—or something to its effect—is that we must figure out which `venv` folder to use when the user does not constrain `PY`. We no longer have a generic `py3.venv` folder (see PR description for rationale).

--

We could make this more generic by simply echoing `echo "3${version_number}"`. 

Let me know if you prefer this! The downside I see to this approach is the script wouldn't die if user tried to use python 3.0-3.5.